stories/components/Modal.stories.mdx

```diff
@@ -49,68 +69,261 @@ Nevertheless, the **Modal.Header** will look the same whether you go for option
+                the release of Letraset sheets containing Lorem Ipsum passages, and more recently
+                with desktop publishing software like Aldus PageMaker including versions of Lorem
+                Ipsum.`
+
+export const PlainMediumText = `The Body of a Modal can contain whatever you like! Like this very long Lorem Ipsum  
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
+                Ipsum has been the industrys standard dummy text ever since the 1500s, when an
+                unknown printer took a galley of type and scrambled it to make a type specimen book.
+                It has survived not only five centuries, but also the leap into electronic
+                typesetting, remaining essentially unchanged. It was popularised in the 1960s with
+                the release of Letraset sheets containing Lorem Ipsum passages, and more recently
+                with desktop publishing software like Aldus PageMaker including versions of Lorem
+                Ipsum.`
+
+<!-- Prop type tables -->
+<!--
+Modal story is made up of several components, each with their own arguments,
+so <ArgsTable of={Modal} /> wouldn't work in this case. Instead, we specify the
+props of one story i.e. Playground
+-->
+
+<ArgsTable story="Playground" />
 
 <!-- Stories -->
 
-<Canvas>
-  <Story name="ModalHeaderOnlyStory">
-    <ModalHeaderOnlyStory />
-   </Story>
-</Canvas>
-
-<Canvas>
-  <Story name="ModalHeaderAndBodyStory">
-    <ModalHeaderAndBodyStory />
-   </Story>
-</Canvas>
-
-
-<Canvas>
-  <Story name="ModalHeaderBodyAndActionsStory">
-    <ModalHeaderBodyAndActionsStory />
-   </Story>
-</Canvas>
-
-
-<Canvas>
-  <Story name="ModalScrollableBodyStory">
-    <ModalScrollableBodyStory />
-   </Story>
-</Canvas>
-
-<Canvas>
-  <Story name="PlainMediumModalStory">
-    <PlainMediumModalStory />
-   </Story>
-</Canvas>
-
-<Canvas>
-    <Story name="ModalPlaygroundStory" args={{
-        plain: false,
-        showCloseIcon: false,
-        medium: false,
-        large: false,
-        title: 'Header of Modal',
-        subtitle: 'This is a smaller description',
-    }}>
-     {Template.bind({})}
-   </Story>
-</Canvas>
\ No newline at end of file
+<Preview withToolbar>
+    <Story
+        parameters={{ docs: { source: { type: 'code' } } }}
+        name="Playground"
+        args={{
+            plain: false,
+            showCloseIcon: false,
+            medium: false,
+            large: false,
+            title: 'Header of Modal',
+            subtitle: 'This is a smaller description',
+        }}
+    >
+        {WithPlaygroundTemplate.bind({})}
+    </Story>
+</Preview>
+
+<Preview withToolbar>
+    <Story
+        parameters={{ docs: { source: { type: 'code' } } }}
+        name="Modal Header"
+        args={{
+            title: 'Header of Modal',
+            buttonTitle: 'Header Only',
+            buttonText: 'Click me to launch a Modal with Header',
```

Comment:
I don't think it was like this the first time I tested the stories, but the non-playground examples will now create a modal in the playground area:

![image](https://user-images.githubusercontent.com/8531248/118469033-f1a2e180-b737-11eb-8e3e-0e9802603630.png)

Reply:
Great spot - this was due to the playground story template including an additional div with id `modal_box` which is what the helper uses to render stories. Will be fixed in next commit